### PR TITLE
Update some of the .idea project configuration files

### PR DESCRIPTION
**Pull request by Sigmonia**
#### Rationale
Update some of the .idea files for the new Git world

#### Changes
Change the projects VCS type to Git
Update the root .gitignore file to ignore most of the .idea folder

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,12 @@ server/testAutomation/
 
 # Include the modules build.gradle file -- this line will override previous exclusion
 !server/modules/build.gradle
+
+# Ignore parts of the .idea dir
+.idea/libraries
+.idea/modules
+.idea/.gitignore
```
Comment (labkey-susanh) [inline .gitignore:26]: This is presumably generated by intelliJ.  Interesting that it doesn't think the libraries and modules directories are to be ignored, but whatever.

File: .gitignore [hunk 2]
```diff
@@ -19,3 +19,12 @@ server/testAutomation/
...
+.idea/jarRepositories.xml
+.idea/modules.xml
+.idea/server.iml
```
Comment (labkey-susanh) [inline .gitignore]: I don't have this file.

File: .gitignore [hunk 3]
```diff
@@ -19,3 +19,12 @@ server/testAutomation/
 
 # Include the modules build.gradle file -- this line will override previous exclusion
 !server/modules/build.gradle
+
+# Ignore parts of the .idea dir
+.idea/libraries
+.idea/modules
```
Comment (labkey-susanh) [inline .gitignore]: Do these two directories need trailing slashes?

Comment (Sigmonia) [inline .gitignore]:
```suggestion
.idea/libraries/
.idea/modules/
```

File: .gitignore (hunk 2 above)
Comment (Sigmonia) [inline .gitignore]:
```suggestion
```

**Review by labkey-susanh — APPROVED** [review]

**Pull request merged**